rero/rero-ebooks
# clean: remove not necessary modules and remove view

How to test:
- ./scripts/bootstrap
- ./scripts/setup
- ./scripts/server
- Check if oai server response
- The base url return "Not Found"

OAI URL Server:
- https://127.0.0.1:5000/oai2d?verb=ListIdentifiers&metadataPrefix=marc21
- https://127.0.0.1:5000/oai2d?verb=ListRecords&metadataPrefix=marc21

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-authored-by: Peter Weber <peter.weber@rero.ch>
